tests/Feature/AbstractRequestTest.php

```diff
@@ -31,37 +31,36 @@ class AbstractRequestTest extends BaseTestCase
 {
     use MocksGuzzleInstance;
 
+    protected function setUp(): void
+    {
+        parent::setUp();
+        Storage::fake('api-logs');
+    }
+
     protected function mockRequestWithLog()
     {
         return new class extends ConcreteRequest {
             use ParseResponseJSON;
 
             protected bool $shouldLog = true;
 
-            public $logged;
```

Comment:
I think the idea behind this `public logged` is obsolete since we started adding the getLastLogFile and getLastLogContents, so I made this mock more like the real code by and less snowflakey.